lib/tests/test_ancestry.c

```diff
@@ -2713,6 +2713,54 @@ test_census_event(void)
     tsk_table_collection_free(&tables);
 }
 
+static void
+test_population_activation_event(void)
```

Comment:
Rename to ``test_activate_population_event`` (this will help when you're search-replacing to implement deactivate population). Also, we want another test case called ``test_activate_population_event_errors`` which provokes the error cases identified above.